aten/src/ATen/native/Pow.cpp

```diff
@@ -53,12 +53,26 @@ Tensor& pow_(Tensor& base, Scalar alpha) {
 }
 
 Tensor pow(const Tensor& base, const Tensor& exp) {
-  Tensor result = at::empty({0}, base.options());
```

Comment:
I think this is even generally wrong for real tensors and real exponents as well. see [here](https://github.com/pytorch/pytorch/issues/37673)

Reply:
Exactly, a base with integer type and a float Exp results in integer.
I didn't fix it in here to keep the same semantics, #37098 will fix it